.vscode/settings.json

```diff
@@ -9,14 +9,6 @@
   "yaml.schemaStore.enable": true,
   "yaml.validate": true,
   "yaml.schemas": {
-    "http://json-schema.org/draft-07/schema#": "*.schema.yaml",
-    "src/schemas/pa-yaml/v3.0/pa.schema.yaml": [
-      //"*.pa.yaml",
-      "src/schemas-tests/pa-yaml/v3.0/**/*.pa.yaml"
-    ],
-    "https://raw.githubusercontent.com/microsoft/PowerApps-Tooling/master/schemas/pa-yaml/v3.0/pa.schema.yaml": [
-      "src/schemas-tests/**/*.pa.yaml"
-    ],
-    "https://raw.githubusercontent.com/microsoft/PowerApps-Tooling/master/docs/pa.yaml-schema.json": "src/Persistence.Tests/**/*.pa.yaml"
+    "https://raw.githubusercontent.com/microsoft/PowerApps-Tooling/master/docs/pa.yaml-schema.json": "*.pa.yaml"
```

Comment:
What is the issue that you're having?

All of the yaml files in this repo fall in the folders configured with the previous configuration. Was there a folder missing?

Reply:
each individual should add this line to their own user-level settings.json file. This should be our recommended approach for everyone.

The configuration in here applies to just this repo and explicitly to specific file/folder paths.

It sounds like you want all other *.pa.yaml files (which do not have a schema already defined) to use the v2.2 schema correct?

may I suggest the following, with comments added for clairity:
```json
{
  "cSpell.words": [
    "Dataverse",
    "msapp",
    "PPUX",
    "RGBA"
  ],
  "[yaml]": {},
  "yaml.schemaStore.enable": true,
  "yaml.validate": true,
  "yaml.schemas": {
    // *.schema.yaml files should use JSON schema:
    "http://json-schema.org/draft-07/schema#": "*.schema.yaml",
    // This schema registration supports local development of the v3.0 schema:
    "src/schemas/pa-yaml/v3.0/pa.schema.yaml": [
      "src/schemas-tests/pa-yaml/v3.0/**/*.pa.yaml"
    ],
    // This schema registration applies the checked in master version of the schema to these folders:
    "https://raw.githubusercontent.com/microsoft/PowerApps-Tooling/master/schemas/pa-yaml/v3.0/pa.schema.yaml": [
      "src/schemas-tests/**/*.pa.yaml"
    ],
    // This schema registration is the 'default' for any *.pa.yaml files which are not covered by above registrations:
    "https://raw.githubusercontent.com/microsoft/PowerApps-Tooling/master/docs/pa.yaml-schema.json": [
      "*.pa.yaml", // Set all other pa.yaml files to default to the v2.2 schema
      // Ensure the persistence tests use v2.2 schema even if the repo default ends up changing in the future:
      "src/Persistence.Tests/**/*.pa.yaml"
    ]
  }
}
```

Reply:
I added some of the comments you suggested and "*.schema.yaml"